### ISSUE TITLE
qtwebkit-5.212.0-alpha3 does not build with Qt 5.12.7 on Windows 10 with Visual Studio 2015

### ISSUE THREAD

**Issue by fichte** [qtwebkit/qtwebkit#936]
After successfully compiling Qt 5.12.7 i tried to compile the most recent release Version qtwebkit-5.212.0-alpha3.

According to the documentation i tried to compile qtwebkit with the following command:

perl Tools/Scripts/build-webkit --qt --release --prefix=c:\tools\libs\qt_5_12_7 --cmakeargs="-Wno-dev -DCMAKE_PREFIX_PATH=c:\tools\libs\qt_5_12_7"

With Qt 5.12.5 everything works fine.

The Result was:

```
-- All-in-One build is enabled.
-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Source/WebCore/PlatformQt.cmake
-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Source/WebKit/PlatformQt.cmake
CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/PlatformQt.cmake:347 (qt5_add_resources)
  Source/cmake/WebKitMacros.cmake:5 (include)
  Source/WebKit/CMakeLists.txt:41 (WEBKIT_INCLUDE_CONFIG_FILES_IF_EXISTS)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit/qt/tests/CMakeLists.txt:54 (qt5_add_resources)


-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Source/WebKit2/PlatformQt.cmake
CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit2/PlatformQt.cmake:217 (qt5_add_resources)
  Source/cmake/WebKitMacros.cmake:5 (include)
  Source/WebKit2/CMakeLists.txt:785 (WEBKIT_INCLUDE_CONFIG_FILES_IF_EXISTS)


CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Source/WebKit2/UIProcess/API/qt/tests/CMakeLists.txt:78 (qt5_add_resources)


-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Source/PlatformQt.cmake
-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Tools/PlatformQt.cmake
-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Tools/DumpRenderTree/PlatformQt.cmake
CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Tools/DumpRenderTree/PlatformQt.cmake:30 (qt5_add_resources)
  Source/cmake/WebKitMacros.cmake:5 (include)
  Tools/DumpRenderTree/CMakeLists.txt:115 (WEBKIT_INCLUDE_CONFIG_FILES_IF_EXISTS)


-- Using platform-specific CMakeLists: C:/tools/libs/src/qtwebkit-5.212.0-alpha3/Tools/ImageDiff/PlatformQt.cmake
CMake Error at C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:211 (configure_file):
  configure_file Problem configuring file
Call Stack (most recent call first):
  C:/tools/libs/qt_5_12_7/lib/cmake/Qt5Core/Qt5CoreMacros.cmake:281 (_QT5_PARSE_QRC_FILE)
  Tools/MiniBrowser/qt/CMakeLists.txt:42 (qt5_add_resources)
```

**Comment by annulen** [comment]
Reproduced

**Comment by annulen** [comment]
Could you check if reverting change at https://codereview.qt-project.org/c/qt/qtbase/+/282506/3/src/corelib/Qt5CoreMacros.cmake#b62 fixes the problem?

**Comment by annulen** [comment]
Looks like it was https://bugreports.qt.io/browse/QTBUG-81715

**Comment by annulen** [comment]
Fixed in 5.12.8 branch of Qt, release is coming soon.
